tangelo/algorithms/classical/semi_empirical_solver.py

```diff
@@ -62,7 +62,7 @@ def simulate(self):
         """
         from pyscf.semiempirical import mindo3
 
-        solver = mindo3.RMINDO3(self.molecule.to_pyscf()).run(verbose=0)
+        solver = mindo3.RMINDO3(self.molecule.solver.to_pyscf(self.molecule)).run(verbose=0)
```

Comment:
See other comments but that call feels non-sensical to me. I think we need to make a design change.